content/pages/education/apply.md

```diff
@@ -2,11 +2,15 @@
 layout: page-breadcrumbs.html
 template: detail-page
 title: Education Benefits Application Process
-includes_edu_wizard: true
 display_title: Application Process
 plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
 collection: education
 order: 2
+widgets:
+  - root: react-applicationStatus
+    timeout: 20
+    loadingMessage: Checking your application status.
+    errorMessage: <strong>We're sorry. Something went wrong when we tried to load your saved application.</strong><br/>Please try refreshing your browser in a few minutes.
 description: Find the right VA education benefits for you, and apply to start getting help paying tuition. We can also help you find the right school or training program.
```

Comment:
@peggygannon could this be simplified by rephrasing as "..., and apply for tuition assistance." Or is "tuition assistance" not an accurate way to express "help paying tuition" or is it perhaps less clear to users?